app/Exceptions/Handler.php

```diff
@@ -154,9 +149,12 @@ private function reportWithSentry($e)
             ];
         }
 
-        Sentry::user_context($userContext);
+        app('sentry')->configureScope(function ($scope) use ($e, $userContext) {
+            $scope->setUser($userContext);
+            $scope->setTag('http_code', $this->statusCode($e));
```

Comment:
should have an explicit cast to `string` since `statusCode` should return `int` only